corim/signedcorim.go

```diff
@@ -15,7 +16,8 @@ import (
 )
 
 var (
-	ContentType          = "application/rim+cbor"
+	ContentType          = "application/unsigned-corim+cbor"
```

Comment:
The media types defined by CoRIM are:

* [application/corim-signed+cbor](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#sec-mt-corim-signed)
* [`application/corim-unsigned+cbor`](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#sec-mt-corim-unsigned)

Reply:
Fixed.